components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/IdentityProviderManager.java

```diff
@@ -32,6 +32,7 @@
 import org.wso2.carbon.identity.application.common.model.*;
```

Comment:
remove star imports. Even if you are not the one who did it, it's good if you can fix this